### PR TITLE
Create ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3974](https://cumulocity.atlassian.net/browse/DM-3974)
Original PR: [7224](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7224)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Pass selected ids to bulk action control
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]: The Claude AI suggested to use the following title:
```suggestion
title: Pass selected IDs to bulk action control
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3974
+version: 1020.31.0
+---
+Pass selected ids to bulk action control
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]: The Claude AI suggested to use the following body:
```suggestion
In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to perform bulk actions on a subset of items. With this change, the selected IDs are now passed to the bulk action control. This allows users to select specific items in a list and perform bulk actions only on those selected items, enabling more granular control over bulk operations.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Pass selected ids to bulk action control
+product_area: Device management & connectivity
+change_type:
+  - value: change-QHu1GdukP
+    label: Feature
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
  - value: component-YbYJ3gLU_
    label: Web SDK
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md (hunk 1 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
title: 'showIf' function of bulk action controls receives an array of selected items' IDs
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md (hunk 2 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to display a bulk action conditionally based on the subset of selected items. With this change, the selected IDs are now passed to the bulk action control 'showIf' function.
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: 'showIf' function of bulk action controls receives an array of selected items' IDs
+product_area: Device management & connectivity
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
product_area: Application enablement & solutions
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3974
+version: 1020.31.0
+---
+In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to display a bulk action conditionally based on the subset of selected items. With this change, the selected IDs are now passed to the bulk action control 'showIf' function.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to display a bulk action conditionally based on the subset of selected items. With this change, the selected IDs are now passed to the bulk action control `showIf` function.
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: 'showIf' function of bulk action controls receives an array of selected items' IDs
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
title: Selected item IDs are now passed to the bulk action control
```

File: content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3974
+version: 1020.31.0
+---
+In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to display a bulk action conditionally based on the subset of selected items. With this change, the selected IDs are now passed to the bulk action control `showIf` function.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-31-0-pass-selected-ids-to-bulk-action-control.md]:
```suggestion
In the past, the bulk action control did not receive the IDs of the selected items, which made it impossible to display a bulk action conditionally based on the subset of selected items. With this change, the selected item IDs are now passed to the bulk action control `showIf` function.
```

**Comment by BeateRixen** [comment]
@bozhidardedov This is still very much from the developer perspective. What was the issue that users had and what is now working? Users probably don´t know much about bulk "bulk action control". Could you please try to rephrase?

**Comment by bozhidardedov** [comment]
> @bozhidardedov This is still very much from the developer perspective. What was the issue that users had and what is now working? Users probably don´t know much about bulk "bulk action control". Could you please try to rephrase?

@BeateRixen It's only a change in the Web SDK with no direct effect for end users. It's oriented at developers using data grid and implementing bulk action controls.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
> > @bozhidardedov This is still very much from the developer perspective. What was the issue that users had and what is now working? Users probably don´t know much about bulk "bulk action control". Could you please try to rephrase?
> 
> @BeateRixen It's only a change in the Web SDK with no direct effect for end users. It's oriented at developers using data grid and implementing bulk action controls.

Please approve and merge :-)

**Review by dafkk — APPROVED** [review]

**Pull request merged**